src/client.d.ts

```diff
@@ -1,194 +1,160 @@
-declare module '@mistralai/mistralai' {
-    export interface ModelPermission {
-        id: string;
-        object: 'model_permission';
-        created: number;
-        allow_create_engine: boolean;
-        allow_sampling: boolean;
-        allow_logprobs: boolean;
-        allow_search_indices: boolean;
-        allow_view: boolean;
-        allow_fine_tuning: boolean;
-        organization: string;
-        group: string | null;
-        is_blocking: boolean;
-    }
-
-    export interface Model {
-        id: string;
-        object: 'model';
-        created: number;
-        owned_by: string;
-        root: string | null;
-        parent: string | null;
-        permission: ModelPermission[];
-    }
-
-    export interface ListModelsResponse {
-        object: 'list';
-        data: Model[];
-    }
-
-    export interface Function {
-        name: string;
-        description: string;
-        parameters: object;
-    }
-
-    export enum ToolType {
-        function = 'function',
-    }
-
-    export interface FunctionCall {
-        name: string;
-        arguments: string;
-    }
-
-    export interface ToolCalls {
-        id: 'null';
-        type: ToolType = ToolType.function;
-        function: FunctionCall;
```

Comment:
What happened here? :)

Reply:
What do you mean?

Reply:
```
id: 'null';
type: ToolType = ToolType.function;
```

As you mentioned, the enums don't work at runtime, and `null` is set to match the string null, not the null *type* (which is actually not appropriate either). Also, it's trying to set a default (`ToolType = ToolType.function;`) inside a `.d.ts` file, which I don't believe actually works. 

More a rhetorical question than serious :) Wondering how the ..

Reply:
Yeah the `null` string is expected, as before our API change the id returned by the API was always `'null'` (the string). Now it is function call ids yes. The default stuff well I'm not sure ahah